src/sentry/integrations/vercel/integration.py

```diff
@@ -238,22 +234,34 @@ def update_organization_config(self, data):
                 "SENTRY_ORG": {"type": "plain", "value": sentry_project.organization.slug},
                 "SENTRY_PROJECT": {"type": "plain", "value": sentry_project.slug},
                 dsn_env_name: {"type": "plain", "value": sentry_project_dsn},
-                "SENTRY_AUTH_TOKEN": {"type": "secret", "value": sentry_auth_token},
+                "SENTRY_AUTH_TOKEN": {
+                    "type": "secret",
+                    "value": self.get_sentry_auth_token_value(sentry_project, vercel_client),
+                },
                 "VERCEL_GIT_COMMIT_SHA": {"type": "system", "value": "VERCEL_GIT_COMMIT_SHA"},
             }
 
             for env_var, details in env_var_map.items():
-                if details["type"] == "secret":
```

Comment:
I'm a little fuzzy on this but you don't need this check anymore since it's only used for the auth token env var yeah?

Reply:
@ceorourke yeah exactly, instead of updating the `value` in this for loop, I figured we could just get the `value` when we create the `env_var_map`. 

If we ever need to make more env vars that are also secret, we could probs just update the method to take the `secret_name` as well and it make it a more generic `get_secret_value`.